lib/PrometheusPlugin.ts

```diff
@@ -140,8 +137,10 @@ export class PrometheusPlugin extends Plugin {
    * @param {PluginContext}       context - Kuzzle plugin context
    */
   async init (config: PrometheusPluginConfiguration, context: PluginContext) {
-    this.config = _.merge(this.config, config);
     this.context = context;
+    this.config = _.merge(this.config, config);
+    // TODO: remove this.context.accessors.nodeId when Kuzzle Plugin Prometheus v5 is released
```

Comment:
Why do we need to wait this ?

Reply:
Because for now the Kuzzle minimum compatible version is `2.16.0`. But replacing `global.kuzzle.nodeId` by `this.context.accessors.nodeId` will raise this version to `2.16.9`

Reply:
Bumping minimum required Kuzzle version for the plugin should be considered as a good practice and in fact there is no breaking change in doing this as long as you don't update to a new major version of Kuzzle.

On other plugins, Kuzzle minimum version is updated with almost every new release for example

Reply:
Ok good to know I'll update Kuzzle version requirements 😉